test/KnormPostgres.spec.js

```diff
@@ -270,6 +270,15 @@ describe('KnormPostgres', () => {
             expect(forSave, 'was called on', instance);
           });
         });
+
+        it(`does not stringify a sting`, () => {
+          const field = new Field({ name: 'foo', model: Model, type: 'json' });
+          expect(
+            field.cast('hello there', null, { forSave: true }),
+            'to equal',
+            'hello there'
```

Comment:
hmm.. this should be `"hello there"` since that's the stringified version of `'hello there'`. I guess the test should be "does not stringify a stringified string" and the code would do something like:

```js
try {
  JSON.parse(value);
  return; // already stringified, do nothing
} catch (e) {
  return JSON.stringify(value); // not stringified, attempt to stringify
}
```

but that `JSON.parse` will probably cause some performance degradation